chalice/deploy/deployer.py

```diff
@@ -324,18 +325,19 @@ def _first_time_lambda_create(self, config, function_name, stage_name):
             config.project_dir)
         zip_contents = self._osutils.get_file_contents(
             zip_filename, binary=True)
-        return self._aws_client.create_function(
-            function_name, role_arn, zip_contents,
-            config.environment_variables,
-            config.lambda_python_version,
-            self._function_tags(config, stage_name),
-        )
-
-    def _function_tags(self, config, stage_name):
-        # type: (Config, str) -> Dict[str, str]
-        tag = 'version=%s:stage=%s:app=%s' % (chalice_version,
-                                              stage_name, config.app_name)
-        return {'aws-chalice': tag}
+        client_kwargs = {
```

Comment:
That won't work because I have the `create_function` taking in `int` as its type with defaults of 60 and 128 respectively. I chose that in the `create_function` (instead of having a default of `None`) because it was consistent with the `runtime` argument having a default string and not `None`. And `None` cannot be passed into botocore client call because it will not pass validation.

I was going over this prior to your comments and I did not like it as well. One thing that I though of to clean this up was to set defaults in the config object so that if I call `config.lambda_timeout` and `config.lambda_memory_size` I will always at least get at least the default back, which would be an integer. It really helps clean up the logic by hoisting this. What are your thoughts on this?